### ISSUE TITLE
Bug: Selecting text box moves the whole line

### ISSUE THREAD

**Issue by justinegeffen** [mattermost/focalboard#1015]
**Summary:**
When clicking and dragging to select the text area in Meeting notes' table view, it moves the entire text box.

https://user-images.githubusercontent.com/4223530/129629324-29c164af-09dc-4ddd-90c7-30c7aebf644c.mov

**Steps to reproduce the behavior:**
In the Meeting Notes board, select table view. If there's a line with default text, click and drag your cursor as if you want to highlight the text.

**Expected behavior:**
I would expect the text box to be static so I could drag and select all the text I want to replace rather than clicking and having to backspace. 

Alternatively, clicking in the textbox should make the placeholder text disappear. 

**Screenshots:**

**Platform:**
 - Browser and version: Firefox 91
 - OS and version: macOS 11.5

**Comment by chenilim** [comment]
Thanks @justinegeffen! Took a quick look. This repros on FireFox, but not on Chrome. Basically, mouse-based selection doesn't work on table cells. Double-click to select doesn't work either. Mouse-selection does appear to work fine in the card editor though.

@sbishel, assigning to you to reassign. I think mouse text selection in a table cell is a common action, and we should generally investigate why there are bugs in FireFox that don't repro in Chrome. There might be a deeper issue. cc @ogi-m.

**Comment by chenilim** [comment]
This repros on Safari as well, and I've also filed a related issue, #1021 for the property name fields. It appears it might be the same base issue.

**Comment by jespino** [comment]
This is a bit more profund problem, react-dnd solution to fix this problem is to capture the "dragStart" it doesn't allow me to select text. I need to give it another try, but maybe the solution is to add a "handler" at the left of the row to drag and drop from there.

**Comment by chenilim** [comment]
Yup, adding a drag grip on the table's left margin makes sense to me. That's what Notion and other apps do. This might be the right general pattern for draggable elements. What do you think @asaadmahmood?
